src/internal/m365/helper_test.go

```diff
@@ -1,5 +1,15 @@
 package m365
 
+// ---------------------------------------------------------------------------
+// 								>>> TODO <<<
+// This file's functions are duplicated into /drive/helper_test.go, which
```

Comment:
can we get a tech debt ticket for this as well please? That way we have something outside the code that lets us know something needs fixed

Reply:
heh, doubly glad you mentioned this, because I put the comment in the wrong file, too...

https://github.com/alcionai/corso/issues/4846